api/middleware/auth.js

```diff
@@ -11,7 +11,7 @@ const validateToken = async (req, res, next) => {
     next();
   } catch (err) {
     console.error(err);
-    res.status(401).json({ message: "Invalid Access Token" });
+    response.status(401).json({ message: "Invalid Access Token" });
```

Comment:
에러 메세지는 대문자로 수정해 주세요 !
```suggestion
    res.status(401).json({ message: "INVALID_ACCESS_TOKEN" });
```